packages/skill-template/src/engine/index.ts

```diff
@@ -181,7 +183,7 @@ export class SkillEngine {
         },
       },
       ...params,
-      include_reasoning: true,
+      include_reasoning: config?.modelInfo?.capabilities?.reasoning,
     });
```

Comment:
Is there no reasoning here? It would be better not to pass this parameter directly.